python_scripts/nao_server.py

```diff
@@ -92,7 +92,8 @@ def log_transaction(self,msgType,questionNum,otherInfo):
         self.logFile.write(transaction+"\n")
         self.logFile.flush()
 
-        self.update_session(msgType, questionNum, otherInfo)
+        # IMPORTANT! Commenting this out temporarily
+        # self.update_session(msgType, questionNum, otherInfo)
```

Comment:
@kevjiang Just a heads up, I've temporarily commented this (and one other line) out in nao_server.py. The added logging introduces a few bugs (i.e. the Python script crashes when I run the whole setup, tablet + laptop).